src/game/model/Runner.js

```diff
@@ -202,6 +202,15 @@ export class Runner {
     LOAD: 'load'
   };
 
+  /**
+   * Stops the game, clears container and clears game instance
+   */
+  destroy() {
+    this.stop();
+    this.stopListening();
+    this.containerEl = null;
+    Runner.instance_ = null;
```

Comment:
This is the magic bit.
Instance is kept, which (for some reason) fucks it up and keep running in the background even when moving away.
Removing it on unmount fixes the problem

Reply:
Awesome

Reply:
i knew it! (Self-five ;))